supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+
+Migration compression is specific to Red Hat Virtualization, and is the compression of virtual machine image information while a running virtual machine is moved from one server to another. It uses Xor Based Zero Run Length Encoding (XBZRLE) as the compression mechanism.
+
+LZO compression and LZ4 compression are specific compression algorithms, so they are specific enough.
+
+*Use it*: With caution
+
+
+
+[discrete]
+[[copy-v]]
+==== image:images/yes.png[yes] copy (verb)
+*Description*: Generically, the process of creating a duplicate of a file in a different location.
+
+
+[discrete]
+[[deduplication]]
+==== image:images/yes.png[yes] deduplication (noun)
+*Description*: A feature of virtual disk optimization (VDO) that helps conserve storage space by eliminating multiple copies of duplicate blocks. Instead of writing the same data more than once, VDO detects blocks that are duplicates of an existing block, and records them as a reference to the original block. VDO maintains a mapping from logical block addresses, which are used by the storage layer above VDO, to physical block addresses, which are used by the storage layer under VDO. After deduplication, multiple logical block addresses may be mapped to the same physical block address; these are called shared blocks. Block sharing is invisible to users of the storage, who read and write blocks as they would if VDO were not present. When a shared block is overwritten, a new physical block is allocated for storing the new block data to ensure that other logical block addresses that are mapped to the shared physical block are not modified.
+device	Use block or VDO or local storage device instead of using the vague term "device".
+
+*Use it*: Yes
+
+
+[discrete]
+[[directory]]
+==== image:images/yes.png[yes] directory (noun)
+*Description*: A special type of file in a Linux file system that contains a list of objects and their inodes.
+
+*Use it*: Yes
+
+
+[discrete]
+[[disaster-recovery]]
+==== image:images/yes.png[yes] disaster recovery (adjective)
+*Description*: Related to processes or methods used to ensure data integrity and the operational continuity of an environment or infrastructure in case of malicious or accidental interruption.
+
+[discrete]
+[[disconnected-deployment]]
+==== image:images/caution.png[with caution] disconnected deployment or installation (noun)
+*Description*: An installation or deployment method that has been modified so that access to the internet is not required, for example, for use in an airgapped environment.
+
+*Use it*: With caution - support levels vary, so the exact method and limitations must be explained clearly.
+
+*Incorrect forms*: offline deployment, airgapped deployment
+
+[discrete]
+[[physical-disk]]
+==== image:images/yes.png[yes] physical disk (noun)
```

Comment:
This entry is out of alphabetical order too (and virtual disk below too). Can you do a run through to make sure that the list is in order?